### PR TITLE
Optimize service lb policy update

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

/kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/controller/workload/workload_processor.go [hunk 1]
```diff
@@ -567,32 +574,22 @@ func (p *Processor) updateEndpointOneByOne(serviceId uint32, epsUpdate []cache.E
...
 		return p.updateEndpointOneByOne(serviceId, endpointSlice, toLLb)
 	} else {
-		filtered := []cache.Endpoint{}
-		for _, endpoint := range endpoints {
-			if endpoint.Prio > 0 {
-				filtered = append(filtered, endpoint)
-			}
-		}
+		filtered := slices.Filter(endpointSlice, func(e cache.Endpoint) bool {
+			return e.Prio > 0
```
Comment (LiZhenCheng9527) [inline pkg/controller/workload/workload_processor.go:591]: Can this return value be used as a function error return?

Comment (hzxuzhonghu) [reply pkg/controller/workload/workload_processor.go:591]: what do you mean

Comment (LiZhenCheng9527) [reply pkg/controller/workload/workload_processor.go:591]: My mistake. This one is right.

**Pull request merged**